settings/src/components/webauthn/sjinks-utilities.js

```diff
@@ -0,0 +1,78 @@
+/**
```

Comment:
I think it's best to keep it in a separate file from our 1st-party code, to have a clean separation between that and our forked/3rd party code. I also like a separate file because of the length and logical grouping of the functions.

It feels to me like it fits better in the `webauthn` folder, since it's only relevant to that provider. I don't feel strongly though, and a new `utilities` folder works for me too 👍🏻 

In that case, should we also move `tests` outside of `components` (in another PR)?

Reply:
👍🏻 I moved the file in 5468a00 and 0f201fd, and opened #210 for doing the out-of-scope changes

Reply:
Thanks!